cro-clib/src/types.rs

```diff
@@ -50,3 +67,32 @@ impl CroResult {
 pub unsafe fn get_string(src: *const c_char) -> String {
     CStr::from_ptr(src).to_string_lossy().into_owned()
 }
+
+/// value: carson unit  for example) 1_0000_0000 carson = 1 cro, 1 carson = 0.0000_0001 cro
+#[repr(C)]
+pub struct CroUtxo {
+    pub address: [u8; 32], // holder
+    pub value: u64,        // amount in carson unit
+    pub valid_from: u64,   // time lock
+}
+
+#[derive(Clone)]
+pub struct CroTx {
+    pub txin: Vec<WitnessedUTxO>, // TxoPointer, TxOut, TxInWitness
+    // contains TxoPointer(in), TxOut(out), attribites(viewkey)
+    pub tx: Tx,
+    /// network-id
+    pub network: u8,
```

Comment:
again this is something duplicating tx attributes

Reply:
i'll modify now.